src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
I can't merge this I don't think - you're not really handling relative paths at all, you're just hacking around it with the substring thing. I'd sooner we handle paths coming in from TsLint vs paths SonarQube's told us we're analysing more robustly in general I think. Will have a ponder.